compiler/src/main/kotlin/io/spine/protodata/backend/CompilerEvents.kt

```diff
@@ -46,12 +46,14 @@ public object CompilerEvents {
      *
      * The resulting sequence is always finite, it's limited by the type set.
      */
-    public fun parse(request: CodeGeneratorRequest): Sequence<EventMessage> {
+    public fun parse(request: PluginProtos.CodeGeneratorRequest): Sequence<EventMessage> {
         val filesToGenerate = request.fileToGenerateList.toSet()
         val files = FileSet.of(request.protoFileList)
         return sequence {
-            files.files()
-                .filter { it.name in filesToGenerate }
+            val (ownFiles, dependencies) = files.files()
```

Comment:
Maybe, that's something which should be reflected in the description for this method?

Reply:
I'm assuming that you thought that `files()` returned a tuple. In actuality, it's the `partition(..)` that chains to this call. I've updated the code layout so that it's more clear.